Admin.API/Program.cs

```diff
@@ -0,0 +1,98 @@
+using Microsoft.Extensions.Options;
+using System.Reflection;
+using Admin.API.Configuration;
+using Admin.API.Extensions;
+using Enmeshed.Tooling.Extensions;
+using Autofac.Extensions.DependencyInjection;
+using Serilog;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.IdentityModel.Logging;
+using Backbone.Modules.Devices.Application;
+
+Log.Logger = new LoggerConfiguration()
+    .WriteTo.Console()
+    .CreateBootstrapLogger();
+
+var builder = WebApplication.CreateBuilder(args);
+builder.WebHost
+    .UseKestrel(options =>
+    {
+        options.AddServerHeader = false;
+        options.Limits.MaxRequestBodySize = 20.Mebibytes();
+    });
+
+LoadConfiguration(builder, args);
+
+builder.Host
+    .UseSerilog((context, configuration) => configuration.ReadFrom.Configuration(context.Configuration))
+    .UseServiceProviderFactory(new AutofacServiceProviderFactory());
+
+ConfigureServices(builder.Services, builder.Configuration, builder.Environment);
+
+var app = builder.Build();
+Configure(app);
+
+app.Run();
+
+static void ConfigureServices(IServiceCollection services, IConfiguration configuration, IHostEnvironment environment)
+{
+    services
+        .ConfigureAndValidate<AdminConfiguration>(configuration.Bind)
+        .ConfigureAndValidate<ApplicationOptions>(options => configuration.GetSection("Modules:Devices:Application").Bind(options));
+
+#pragma warning disable ASP0000 We retrieve the Configuration via IOptions here so that it is validated
+    var parsedConfiguration = services.BuildServiceProvider().GetRequiredService<IOptions<AdminConfiguration>>().Value;
+#pragma warning restore ASP0000
+
+    services.AddCustomSwaggerWithUi();
```

Comment:
Just noticed that you are missing the following custom extension methods from Backbone.API:
* AddCustomFluentValidation
* AddCustomAspNetCore
* 
Is there a reason why you didn't add these?

Reply:
I added those initially, but I tried to strip down as much as I could whithout breaking the API functionality.
Should I include those two calls?

Reply:
Yep. The first one is for proper validation and the second one sets up AspNetCore in the "correct" way (you should remove the calls to AddAuthentication and AddAuthorization from the copied code, of course)